docs/resources/cluster.md

```diff
@@ -560,6 +560,8 @@ The following attributes are exported:
 
 ### `rke_config`
 
+**Note** `rke_config` works the same as within Rancher GUI, thus it will not _provision_ hosts when using `cloud_provider` custom. Provisioning clusters could be done by using the [Terraform provider RKE](https://github.com/rancher/terraform-provider-rke).
+
```

Comment:
`...thus it will not _provision_ hosts when using cloud_provider custom...` may be is more clear to say that it's not deploying nodes if no cloud provider is configured?? 

`Provisioning clusters could be done by using the [Terraform provider RKE](https://github.com/rancher/terraform-provider-rke).` I thinks this is confusing as cluster can be provisioned from this provider, and you are not getting same result using RKE provider. I mean, using RKE and importing the cluster to Rancher would be an option, but not doing same task, you are not generating a custom cluster. I think a clarification saying that on custom clusters nodes are not deployed/configured automatically and it's expected to be already up and ready to execute the node_command, would be more proper.